src/Core/DesignTokens/Services/IThemeService.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.Fast.Components.FluentUI.DesignTokens;
+
+public interface IThemeService
```

Comment:
Why to have properties Read Only but updatable via methods?
And not...
```csharp
public interface IThemeService
{
    StandardLuminance Theme { get; set; }
    OfficeColor AccentColor { get; set; }
    LocalizationDirection Direction { get; set; }
    Task SetDarkModeAsync(bool dark);
}
```

Reply:
I have exposed the properties on the intention of providing users an easier option to bind it to controls and components.

We have a 2 option here, either we keep the properties and add a setter and do the theme and other changes using property only or else we can remove the read-only properties and add method to get the current theme and others

Reply:
I think it's best to have Get/Set properties. If possible.
Another "detail" is probably to convert the OfficeColor to a class with constants (ex. `OfficeColor.Access = "#a4373a"`). You can therefore use ´AccentColor` as a string. 
In this way, the user can also use the color of his choice, or take predefined colors from this list of constants.
@vnbaaij what is your opinion?

Reply:
> Why to have properties Read Only but updatable via methods?
> And not...
> ```csharp
> public interface IThemeService
> {
>     StandardLuminance Theme { get; set; }
>     OfficeColor AccentColor { get; set; }
>     LocalizationDirection Direction { get; set; }
>     Task SetDarkModeAsync(bool dark);
> }
> ```

We might need to go with the Method route since we are using asynchronous functions here to change the theme, accent color, etc..

In my opinion, it's best to keep the read-only property to get the current/selected Theme, color, etc.. and methods for changing themes etc..